packages/e2e-tests/specs/editor/blocks/image.test.js

```diff
@@ -337,15 +341,15 @@ describe( 'Image', () => {
 		await editButton.click();
 
 		await page.waitForSelector( '.block-editor-url-input__input' );
-		await page.evaluate(
-			() =>
-				( document.querySelector(
-					'.block-editor-url-input__input'
-				).value = '' )
-		);
 
+		// Clear the input field. Delay added to account for typing delays.
+		const inputField = await page.$( '.block-editor-url-input__input' );
+		await inputField.click( { clickCount: 3, delay: 100 } );
+		await page.keyboard.press( 'Backspace', { delay: 100 } );
```

Comment:
Keeping the total delay overall the same but just using all of it to ensure the selection of the input field text has the e2es passing more reliably for me.

```suggestion
		await inputField.click( { clickCount: 3, delay: 200 } );
```